.github/CODEOWNERS

```
@@ -159,6 +159,7 @@ receiver/activedirectorydsreceiver/                      @open-telemetry/collect
 receiver/aerospikereceiver/                              @open-telemetry/collector-contrib-approvers @djaglowski @antonblock
 receiver/apachereceiver/                                 @open-telemetry/collector-contrib-approvers @djaglowski
 receiver/awscloudwatchreceiver/                          @open-telemetry/collector-contrib-approvers @djaglowski @schmikei
+receiver/awscloudwatchmetricsreceiver/                   @open-telemetry/collector-contrib-approvers @lewis262626 @jpkrohling @kovrus
```

```suggestion
receiver/awscloudwatchmetricsreceiver/                   @open-telemetry/collector-contrib-approvers @jpkrohling @kovrus
```

Given that you're not a member of OpenTelemetry yet, we cannot list you as codeowner. Please see https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/20868 as background.

An easy way to go about this is to remove your name for the time being, merge this in and use me and the sponsor of the component @jpkrohling as sponsors to get you added as member of the OpenTelemetry organization.